src/components/Records/Search/Output/RecordsCardColumn.vue

```diff
@@ -114,137 +114,153 @@
-                record['reverseRecordAssociations'].forEach(function (association) {
-                    records[association['fairsharingRecord'].registry].val += 1
-                });
-                return records;
-            },
-            setChips: function (record) {
-                let node;
-                for (node in record) {
-                    if (node === 'subjects' || node === 'domains' || node === 'taxonomies' || node === 'userDefinedTags') {
-                        this.organizeChips(record, node);
-                    }
-                }
-            },
-            organizeChips(record, node) {
-                this.organizeButtons(record, node);
-                for (const [key, value] of Object.entries(record[node])) {
-                    if (key > 2) {
-                        break;
-                    }
-                    this.Chips[node][key] = {label: value.label, active: false};
-                }
-            },
-            organizeButtons(record, node) {
-                if (record[node].length > 0 && this.allowLoop) {
-                    this.allowLoop = false;
-                    this.currentActiveChips = node
-                    this.buttons.map(item => {
-                        if (item.title === node) {
-                            item.active = true
-                        }
-                    });
-                }
-            }
-            /*
-                        scrollToTop: function () {
-                            let myDiv = document.getElementById('scroll-target');
-                            myDiv.scrollTo(0, 0);
-                        }
-            */
+        collection: {
+          val: 0,
+          label: "collections"
         },
+      };
+      record['recordAssociations'].forEach(function (association) {
+        records[association['linkedRecord'].registry].val += 1
+      });
+      record['reverseRecordAssociations'].forEach(function (association) {
+        records[association['fairsharingRecord'].registry].val += 1
+      });
+      return records;
+    },
+    setChips: function (record) {
+      let node;
+      for (node in record) {
+        if (node === 'subjects' || node === 'domains' || node === 'taxonomies' || node === 'userDefinedTags') {
+          this.organizeChips(record, node);
+        }
+      }
+    },
+    organizeChips(record, node) {
+      this.organizeButtons(record, node);
+      for (const [key, value] of Object.entries(record[node])) {
+        if (key > 2) {
+          break;
+        }
+        this.Chips[node][key] = {label: value.label, active: false};
+      }
+    },
+    organizeButtons(record, node) {
+      if (record[node].length > 0 && this.allowLoop) {
+        this.allowLoop = false;
+        this.currentActiveChips = node
+        this.buttons.map(item => {
+          if (item.title === node) {
+            item.active = true
+          }
+        });
+      }
+    },
+    getActualButtonTitle(ItemTitle) {
```

Comment:
rename getButtonLabel.
This method is shared by the two component so refactor it into a mixin.